integration-tests-ex/cases/cluster/Common/dependencies.yaml

```diff
@@ -67,7 +67,7 @@ services:
   # See https://hub.docker.com/_/mysql
   # The image will intialize the user and DB upon first start.
   metadata:
-    #    platform: linux/x86_64  - Add when running on M1 Macs
+    platform: linux/x86_64  #- Add when running on M1 Macs
```

Comment:
Should this be checked in?

Reply:
idk, i guess it seemed to not make anything fail, and it was sort of annoying to have to hunt this down to make the new integration tests work on an M1 mac... but I can revert just in case